stix_shifter_modules/crowdstrike/stix_transmission/connector.py

```diff
@@ -195,10 +230,13 @@ async def create_results_connection(self, query, offset, length):
             if not return_obj.get('success'):
                 return_obj['success'] = True
             return return_obj
-
+        except QueryException as ex:
+            return ex.args[0]
         except Exception as ex:
-            if response_txt is not None:
-                ErrorResponder.fill_error(return_obj, message='unexpected exception', connector=self.connector)
-                self.logger.error('can not parse response: ' + str(response_txt))
-            else:
-                raise ex
+            error_dict = {}
+            error_dict['type'] = 'AttributeError'
+            error_dict['message'] = 'Error while parsing API response: ' + str(ex)
```

Comment:
connector name?

Reply:
ErrorResponder.fill_error() function ingest connector name for all error response